open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/handlers/SubjectAreaGlossaryHandler.java

```diff
@@ -340,4 +330,40 @@ private void updateAttributes(Glossary oldGlossary, Glossary newGlossary) {
         }
         return response;
     }
+
+    /**
+     * Get Glossary terms
```

Comment:
I suggest the description and method name be enhanced to say -get glossary terms for this category